uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
```

Comment:
> I just did some searching and I can't find out why this is here, it's probably cruft.

I ought to have left a code comment for later people, since this is definitely non-obvious.

We have a ``@cached_property`` decorator in ``decorators.py`` which lets us define properties as follows

```python
class Foo:
    @cached_property
    def bar(self):
        return 5
```

So what's the difference between ``@property`` and ``@cached_property``?  The latter (as its name implies) caches the result locally.  It's equivalent to saying

```python
class Foo:
    @property
    def bar(self):
        if not hasattr(self, '_bar'):
            self._bar = 5
        return self._bar
```

In other words, the first time we access the property its value is computed and stored, and every other time we just return the cached value.  (This is a dumb example because ``.bar`` is always just ``5`` but you get the idea.)  This can be dangerous, so we need to be careful to NOT use ``@cached_property`` in cases where the value might change over the lifetime of the object instance.

So if we look at that implementation, one thing it allows us to do is pre-load the cached value.  Like, if we have a collection of ``Foo`` instances, and we know the ``.bar`` values for each of them, we can iterate over them and set ``foo._bar`` on each of them.  That's what we're doing here when we set ``job._available_staffers`` on all of the ``Job`` instances in this function.